### ISSUE TITLE
ObjectCaching.cs error

### ISSUE THREAD

**Issue by maxkratt** [FirstGearGames/FishNet#760]
**General**
Unity version: 2022.3.41f1
Fish-Networking version: 4.4.2
Discord links: 
https://discord.com/channels/424284635074134018/1275724816770400300/1275724816770400300
https://discord.com/channels/424284635074134018/1275732707984539702/1275740966183108683

**Description**
After removing the old version and importing the new version the following compile time errors are show:
```
Assets/FishNet/Runtime/Managing/Client/Object/ObjectCaching.cs(286,106): error CS1061: 'CachedNetworkObject' does not contain a definition for 'SceneName' and no accessible extension method 'SceneName' accepting a first argument of type 'CachedNetworkObject' could be found (are missing a using directive or an assembly reference?)

Assets/FishNet/Runtime/Managing/Client/Object/ObjectCaching.cs(286,122): error CS1061: 'CachedNetworkObject' does not contain a definition for 'ObjectName' and no accessible extension method 'ObjectName' accepting a first argument of type 'CachedNetworkObject' could be found (are missing a using directive or an assembly reference?)
```

**Comment by Alfa24T** [comment]
Line 586 ObjectCaching.cs
if DEVELOPMENT
        public string SceneName = string.Empty;
        public string ObjectName = string.Empty;
endif

**Comment by umfahrem** [comment]
Seems that DEVELOPMENT #if statements were erroneously removed, since sceneName and objectName are only used for logging purposes. Error probably originated at removal of if statement in [MangedObjects.Spawning.cs](https://github.com/FirstGearGames/FishNet/blob/539765821bd5df4368549d1260e267a2caf54550/Assets/FishNet/Runtime/Managing/Object/ManagedObjects.Spawning.cs#L294) and cascaded upwards.

**Comment by Alfa24T** [comment]
also a lot of Observer RPC debug messages

Got observerRpc
UnityEngine.Debug:Log (object)
FishNet.Component.Transforming.NetworkTransform:RpcLogic___ObserversUpdateClientAuthoritativeTransform_2713644489 (System.ArraySegment`1<byte>,FishNet.Transporting.Channel) (at Assets/FishNet/Runtime/Generated/Component/NetworkTransform/NetworkTransform.cs:2136)
FishNet.Component.Transforming.NetworkTransform:RpcReader___Observers_ObserversUpdateClientAuthoritativeTransform_2713644489 (FishNet.Serializing.PooledReader,FishNet.Transporting.Channel)
FishNet.Object.NetworkBehaviour:ReadObserversRpc (System.Nullable`1<uint>,FishNet.Serializing.PooledReader,FishNet.Transporting.Channel) (at Assets/FishNet/Runtime/Object/NetworkBehaviour.RPCs.cs:205)
FishNet.Managing.Client.ClientObjects:ParseRpcLink (FishNet.Serializing.PooledReader,uint16,FishNet.Transporting.Channel) (at Assets/FishNet/Runtime/Managing/Client/Object/ClientObjects.RpcLinks.cs:58)
FishNet.Managing.Client.ClientManager:ParseReader (FishNet.Serializing.PooledReader,FishNet.Transporting.Channel,bool) (at Assets/FishNet/Runtime/Managing/Client/ClientManager.cs:455)
FishNet.Managing.Client.ClientManager:ParseReceived (FishNet.Transporting.ClientReceivedDataArgs) (at Assets/FishNet/Runtime/Managing/Client/ClientManager.cs:384)
FishNet.Managing.Client.ClientManager:Transport_OnClientReceivedData (FishNet.Transporting.ClientReceivedDataArgs) (at Assets/FishNet/Runtime/Managing/Client/ClientManager.cs:342)
FishNet.Transporting.Tugboat.Tugboat:HandleClientReceivedDataArgs (FishNet.Transporting.ClientReceivedDataArgs) (at Assets/FishNet/Runtime/Transporting/Transports/Tugboat/Tugboat.cs:238)
FishNet.Transporting.Tugboat.Client.ClientSocket:IterateIncoming () (at Assets/FishNet/Runtime/Transporting/Transports/Tugboat/Core/ClientSocket.cs:295)
FishNet.Transporting.Tugboat.Tugboat:IterateIncoming (bool) (at Assets/FishNet/Runtime/Transporting/Transports/Tugboat/Tugboat.cs:211)
FishNet.Managing.Transporting.TransportManager:IterateIncoming (bool) (at Assets/FishNet/Runtime/Managing/Transporting/TransportManager.cs:705)
FishNet.Managing.Timing.TimeManager:TryIterateData (bool) (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:1043)
FishNet.Managing.Timing.TimeManager:IncreaseTick () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:708)
FishNet.Managing.Timing.TimeManager:<TickUpdate>g__MethodLogic|100_0 () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:376)
FishNet.Managing.Timing.TimeManager:TickUpdate () (at Assets/FishNet/Runtime/Managing/Timing/TimeManager.cs:366)
FishNet.Transporting.NetworkReaderLoop:Update () (at Assets/FishNet/Runtime/Transporting/NetworkReaderLoop.cs:28)

**Comment by FirstGearGames** [comment]
Ah the NT, yes you can delete the NT debug log.

**Comment by FirstGearGames** [comment]
Resolved in 4.4.5
